app/exporters/patients_exporter.rb

```diff
@@ -31,10 +31,6 @@ def self.csv(patients)
           :address,
           :medical_history,
           :current_prescription_drugs,
-          :latest_bp_passports,
```

Comment:
I think we might still be able to `includes` the unordered versions of `bp_passports`, `appointments`, `blood_pressures` and `blood_sugars`. wdyt?

Reply:
I think we could. However, to minimize the number of changes we make while fixing this bug, I figured we'd get things back to where they were (no working query optimizations) before we take another look. Wdyt?

Reply:
I think it's ok, I'm only suggesting it because the line list is notoriously slow and getting rid of these might exacerbate it – we could consider waiting and watching, but it is an actively used feature.

I'll leave it up to you to make a call.

Reply:
I think it's ok, I'm only suggesting it because the line list is notoriously slow and getting rid of these might exacerbate it – we could consider waiting and watching, but it is an actively used feature.

I'll leave it up to you to make a call.

Reply:
The tradeoff for speed is memory usage, so I don't wanna make those changes just yet. 👍

Reply:
Makes sense, I guess we need a deeper reconstruction of this mechanism, simple trade-offs don't apply.